tasks/run_agent.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# shellcheck disable=SC2154,SC2034,SC1090
+
+declare PT__installdir
+source "$PT__installdir/deploy_pe/files/common.sh"
+PUPPET_BIN=/opt/puppetlabs/bin
+
+(( EUID == 0 )) || fail "This utility must be run as root"
+
+lockfile="$("${PUPPET_BIN}/puppet" config print agent_catalog_run_lockfile)"
+retries=5
```

Comment:
This seems like a really useful task. If we use this in more places, we may want to consider making the `retries` and `wait_timeout` be configurable then.

Reply:
Mind if we just slap a `TODO` on it?  :smile:

Reply:
Yeah, this is a nonissue, feature request.